inst/extdata/motrpac_colors_abbr.R

```diff
@@ -150,81 +156,86 @@ tissue_cols = c('paxgene rna'='#d92c04', # blood and paxgene rna are the same (R
                 'blood-rna'='#d92c04',
                 'blood_rna'='#d92c04',
                 BLOOD='#d92c04',
-                "t30-blood-rna"='#d92c04',
-                'edta plasma'="#f1997c",
-                'edta-plasma'="#f1997c",
-                'edta_plasma'="#f1997c",
-                plasma="#f1997c",
-                PLASMA="#f1997c",
-                "t31-plasma"="#f1997c", 
+                't30-blood-rna'='#d92c04',
+                'edta plasma'='#f1997c',
+                'edta-plasma'='#f1997c',
+                'edta_plasma'='#f1997c',
+                plasma='#f1997c',
+                PLASMA='#f1997c',
+                't31-plasma'='#f1997c', 
                 heart='#f28b2f',
                 HEART='#f28b2f',
-                "t58-heart"='#f28b2f',
-                aorta="#a5415b",
-                'vena cava'="#a5415b",
-                'vena-cava'="#a5415b",
-                'vena_cava'="#a5415b",
-                VENACV="#a5415b",
-                "t65-aorta"="#a5415b",
-                "t65-vena-cava"="#a5415b",
-                spleen="#f3c288",
-                SPLEEN="#f3c288",
-                "t62-spleen"="#f3c288",
+                't58-heart'='#f28b2f',
+                aorta='#a5415b',
+                'vena cava'='#a5415b',
+                'vena-cava'='#a5415b',
+                'vena_cava'='#a5415b',
+                VENACV='#a5415b',
+                't65-aorta'='#a5415b',
+                't99-vena-cava'='#a5415b',
+                spleen='#f3c288',
+                SPLEEN='#f3c288',
+                't62-spleen'='#f3c288',
                 gastrocnemius='#088c03',
                 'SKM-GN'='#088c03',
-                "t55-gastrocnemius"='#088c03',
+                't55-gastrocnemius'='#088c03',
                 'vastus lateralis'='#025939',
                 'vastus-lateralis'='#025939',
                 'vastus_lateralis'='#025939',
                 'SKM-VL'='#025939',
-                "t56-vastus-lateralis"='#025939',
+                lateralgastroc = '#025939',
```

Comment:
this should be '#025950', not '#025939'. would suggest moving it to the line below so lines are grouped by tissue

Reply:
Great! I will do all that. Thanks a lot!